### PR TITLE
[fix(card-css): css fixes for cards

### PR DESCRIPTION
**Pull request by mokandre**

### CODE REVIEW

File: packages/node_modules/@webex/react-component-adaptive-card/src/adaptiveCard.scss [hunk 1]
```diff
@@ -150,6 +150,31 @@
...
+    position: absolute;
+    height: 100%;
+    top: 0;
+    width: 2.75rem
+    }
+    .md-input__after{
+    right: 0;
+    left: auto;
+    }
+  }
```
Comment (adamweeks) [inline packages/node_modules/@webex/react-component-adaptive-card/src/adaptiveCard.scss:177]: Can you please adjust the indentations of these? The alignment is not correct.

**Review by adamweeks — CHANGES_REQUESTED** [review]

File: packages/node_modules/@webex/react-component-adaptive-card/src/adaptiveCard.scss (hunk 1 above)
Comment (mokandre) [reply packages/node_modules/@webex/react-component-adaptive-card/src/adaptiveCard.scss]: @adamweeks added indentations

**Comment by adamweeks** [comment]
@mokandre code changes look good. Could you please squash your commits into one since each commit will generate a line in the [changelog](https://github.com/webex/react-widgets/blob/master/CHANGELOG.md)?

**Comment by mokandre** [comment]
> @mokandre code changes look good. Could you please squash your commits into one since each commit will generate a line in the [changelog](https://github.com/webex/react-widgets/blob/master/CHANGELOG.md)?

@adamweeks done

**Review by adamweeks — APPROVED** [review]

**Pull request merged**